src/commands/migrate.ts

```diff
@@ -0,0 +1,255 @@
+import Command from "@oclif/command";
```

Comment:
I guess this file should be named something like migrate-service.ts

can you put some lines with the description of what this script is supposed to do as a comment ?

Reply:
sure!